src/registry/partners/preview.tsx

```diff
@@ -0,0 +1,60 @@
+import {PartnersComponentSchema} from '@open-formulieren/types';
+import {FormattedMessage} from 'react-intl';
+
+import {Description} from '@/components/formio';
+import {FieldSet, TextField} from '@/components/formio';
+
+import {ComponentPreviewProps} from '../types';
+
+const Preview: React.FC<ComponentPreviewProps<PartnersComponentSchema>> = ({component}) => {
+  const {key, label, description, tooltip, defaultValue = []} = component;
+
+  return (
+    <FieldSet field={key} label={label} tooltip={tooltip}>
+      <TextField
+        name={`${key}.bsn`}
+        label={<FormattedMessage description="Label for partner's BSN" defaultMessage="BSN" />}
+        value={defaultValue[0].bsn}
```

Comment:
you may use fictive hardcoded preview data here instead of looking into `defaultValue`